### ISSUE TITLE
Drag-and-Drop-Exercises: TypeError undefined object after dropping element

### ISSUE THREAD

**Issue by christianheimann** [tunapanda/h5p-standalone#21]
First of all, thank you Jakeii and to the others working on this h5p standalone project.

Using standalone for many types there is an issue: Drag-and-Drop-Exercises (Type question, not text) do not work properly in a Standalone-Container: When an element is dragged and then dropped, a TypeError occurs:

`TypeError: undefined is not an object (evaluating 'n.data("uiDraggable").originalPosition={top:o.y+"%",left:o.x+"%"}')`

Have a look at the Stack where the error occurs:
<img width="1099" alt="stack" src="https://user-images.githubusercontent.com/12694135/39515969-e3be192a-4dfb-11e8-84be-bd9a36897070.png">

As this works without an error in the standard context, it might be that there is h5p-standalone mixing something up…? Any ideas?

**Comment by Jakeii** [comment]
Hi what version of h5p-standalone are you running? I fix a similar issue/the same issue previously in https://github.com/tunapanda/h5p-standalone/issues/15

**Comment by christianheimann** [comment]
You are right, I didn't run the most recent version. With the current version, the problem does not occur anymore. Thank you.
